tests/dbfs/test_api.py

```diff
@@ -129,17 +129,27 @@ def test_put_file(self, dbfs_api, tmpdir):
         test_file_path = os.path.join(tmpdir.strpath, 'test')
         with open(test_file_path, 'wt') as f:
             f.write('test')
-
         api_mock = dbfs_api.client
         test_handle = 0
         api_mock.create.return_value = {'handle': test_handle}
         dbfs_api.put_file(test_file_path, TEST_DBFS_PATH, True)
 
-        assert api_mock.add_block.call_count == 1
-        assert test_handle == api_mock.add_block.call_args[0][0]
-        assert b64encode(b'test').decode() == api_mock.add_block.call_args[0][1]
-        assert api_mock.close.call_count == 1
-        assert test_handle == api_mock.close.call_args[0][0]
+    # # Files > 2GB should use open, add_block, close stream upload.
+    # def test_put_large_file(self, dbfs_api, tmpdir):
+    #     test_file_path = os.path.join(tmpdir.strpath, 'test')
+    #     with open(test_file_path, 'wt') as f:
+    #         # 2.1 GB file.
+    #         f.write('\0' * 1)
+    #     api_mock = dbfs_api.client
+    #     test_handle = 0
+    #     api_mock.create.return_value = {'handle': test_handle}
+    #     dbfs_api.put_file(test_file_path, TEST_DBFS_PATH, True)
+    #     add_block_expected = math.ceil(2254856676 / (2**20))
+    #     assert True
+    #     # assert api_mock.add_block.call_count == add_block_expected
+    #     # assert test_handle == api_mock.add_block.call_args[0][0]
+    #     # assert api_mock.close.call_count == 1
+    #     # assert test_handle == api_mock.close.call_args[0][0]
```

Comment:
We can keep this test if we override the 2GB constant in `class DbfsApi(object)` to something small (e.g. 1B) for the test. Let's do it if there's an easy and clean way of doing it. Otherwise, we can just remove this commented block.

Reply:
Ah yes! I was explicitly going to ask what you think about this test before requesting a review. Sorry about that. I will have a look if there is a way of doing that. Thanks you for the input.